pkg/block/namespace.go

```diff
@@ -21,7 +21,16 @@ var (
 	ErrInvalidNamespace = errors.New("invalid namespace")
 )
 
-func (s StorageType) String() string {
+func (s StorageType) MatchingAdapterPrefix() string {
```

Comment:
MatchingAdapterPrefix is a bit confusing for me, it basically returns the storage type (BlockstoreType according to the adapter).

Reply:
Changed to `BlockstoreType`